mobile/client.go

```diff
@@ -212,9 +206,6 @@ func (ps *Props) Accept(callback PropVisitor) (err error) {
 		case bool:
 			err = callback.VisitBool(k, v)
 
-		case int:
-			err = callback.VisitInt(k, v)
```

Comment:
So why not keep handling the int case...
```go
		case int:
			err = callback.VisitNumber(k, float64(v))
```

Reply:
hmm... Good question. I did not value it. But, since you raised it. I think we can keep it.

Reply:
I was under the impression that you added this case because you hit a problem because it was missing.